manim/mobject/number_line.py

```diff
@@ -100,6 +100,8 @@ def __init__(
         numbers_to_include=None,
         # temp, because DecimalNumber() needs to be updated
         number_scale_value=0.75,
+        num_decimal_places=0,
+        font_size=24,
```

Comment:
@hydrobeam I added these parameters since the removal of Container results in these kwargs being necessary. I'm unsure what use they provide though, font_size actually has a comment that it does nothing https://github.com/ManimCommunity/manim/pull/1652/files#diff-d7e574b3b2eee1096bcad222c9828cb67bf6a0b1d84ececc6215f536146610c7R113

Reply:
`font_size` was a relic from the 3b1b edition that I left in case we were to incorporate it in the future. It can be removed if it's causing problems since it might be a while before we incorporate `font_size` and when it happens, it can just be added here.

I'd prefer if `decimal_number_config` stays just so that someone can have reasonable control over the `DecimalNumber` mobjects being created. Although i will say that `num_decimal_places` is definitely the most popular value to edit.